web_widget_open_tab/static/src/js/open_tab_widget.esm.js

```diff
@@ -10,21 +8,7 @@ export class OpenTabWidget extends Component {
         ev.stopPropagation();
     }
     _getReference() {
-        var url = window.location.href;
-        var searchParams = new URLSearchParams(url.split("#")[1]);
-        searchParams.set("view_type", "form");
-        searchParams.set("id", this.props.record.data.id);
-        if (
-            !searchParams.has("model") ||
-            searchParams.get("model") !== this.props.record.resModel
-        ) {
-            searchParams.set("model", this.props.record.resModel);
-            searchParams.delete("action");
-        }
-        return url.split("#")[0] + "#" + searchParams.toString();
-    }
-    loadAttrs(ev) {
-        $(ev.target).tooltip();
+        return window.location.pathname + "/" + this.props.record.data.id;
```

Comment:
This is not working properly. Let me explain the case:

1- Access Apps menu
2- Refresh (this way we set the window.location.pathname
3- Access the User menu (do not press F5)
4- PRess the Open Tab widget